reconcile/cna/client.py

```diff
@@ -11,8 +20,46 @@ class CNAClient:
     https://gitlab.cee.redhat.com/service/cna-management/-/blob/main/openapi/openapi.yaml#/
     """
 
-    def __init__(self, ocm_client: OCMBaseClient):
+    def __init__(self, ocm_client: OCMBaseClient, init_metadata: bool = False):
         self._ocm_client = ocm_client
+        self._metadata = self._init_metadata() if init_metadata else None
+
+    def _init_metadata(self) -> dict[AssetType, AssetTypeMetadata]:
+        asset_types_metadata: dict[AssetType, AssetTypeMetadata] = {}
+        for asset_type_ref in self._ocm_client.get(
+            api_path="/api/cna-management/v1/asset_types"
```

Comment:
nit: could we dedupe a lot of these URLs with something like `{CNA_API_V1}/asset_types`?

Reply:
good point. fixed in [a577a1c](https://github.com/app-sre/qontract-reconcile/pull/3056/commits/a577a1cf233e97dc47ea6fc2e2913092dcce737f)